DEPS

```diff
@@ -15,7 +15,7 @@ deps = {
   "vendor/boto": "https://github.com/boto/boto@f7574aa6cc2c819430c1f05e9a1a1a666ef8169b",
   "vendor/python-patch": "https://github.com/brave/python-patch@d8880110be6554686bc08261766538c2926d4e82",
   "vendor/omaha": {
-    "url": "https://github.com/brave/omaha.git@7e78687ac8d8188c70089e96a172c2d6cfe3a3e4",
+    "url": "https://github.com/brave/omaha.git@826cb43721b58d6da989c8486af9bfe0edb99e22",
```

Comment:
I'd like to see this commit merged into `omaha/main`.

Reply:
Was going to do this after CI here passed. But did it now.